### ISSUE TITLE
delete of a config line seems to differ in counting from what's visible

### ISSUE THREAD

**Issue by frosch03** [frosch03/rct#12]
`config`:
```
# URL                               -> Rooms    ID
---------------------------------------------------
1 https://news.ycombinator.com/rss  -> frosch03 [X]
2                                   -> admin    [X]
3 https://www.xkcd.com/rss.xml      -> frosch03 [X]
4 https://blog.fefe.de/rss.xml?html -> frosch03 [X]
```
`del push 2` 
`config`:
```
# URL                               -> Rooms    ID
---------------------------------------------------
1 https://news.ycombinator.com/rss  -> admin    [X]
2                                   -> frosch03 [X]
3 https://blog.fefe.de/rss.xml?html -> frosch03 [X]
```

**Comment by frosch03** [comment]
It seems that the `del push 2` command actually deleted the 3rd line of the push configuration

**Comment by frosch03** [comment]
fixed with PR #18
